src/Libraries/Microsoft.Extensions.Caching.Hybrid/Microsoft.Extensions.Caching.Hybrid.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>Multi-level caching implementation building on and extending IDistributedCache</Description>
     <TargetFrameworks>$(NetCoreTargetFrameworks)$(ConditionalNet462);netstandard2.0;netstandard2.1</TargetFrameworks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <PackageTags>cache;distributedcache;hybrid</PackageTags>
+    <PackageTags>cache;distributedcache;hybrid;hybridcache</PackageTags>
```

Comment:
Is "hybrid" a useful tag?

Reply:
You're right - that's why I added `hybridcache`, but I have now also removed `hybrid` - good shout